ci/ci.py

```diff
@@ -28,8 +28,10 @@ def append_check(name, *cmd, env=None):
     append_check("flake8", "flake8", ".")
 
     env = os.environ.copy()
-    env["MYPYPATH"] = "stubs/"
-    append_check("mypy", "mypy", "tsrc", "--strict", "--ignore-missing-imports", env=env)
+    # mypy does not work on Python 3.4
+    if sys.version_info.minor >= 5:
```

Comment:
skipping a check depending on what is installed on the buildfarm is weird, no?

Maybe put a warning, do the other checks and return 1 anyway